### DISCUSSION TITLE
set-a-static-ip-address-in-home-assistant-os/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#231] [Announcements]
# set-a-static-ip-address-in-home-assistant-os/

Full credit goes to this forum post. I found myself in need of dropping to the CLI for a fresh Home Assistant OS install on top of Proxmox after using the tteck helper scripts to set a static IP. To do this without access to the Home Assistant UI we

https://blog.ktz.me/set-a-static-ip-address-in-home-assistant-os/

**Comment by hollie** [comment]
Thanks for posting this. I was exactly searching for this info.
